src/app/features/event/event-map/event-map-marker.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@angular/core';
+import * as leaflet from 'leaflet';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class EventMapMarkerService {
+  private markerCoordinates: { altitude: number; latitude: number } | undefined;
+
+  passCoordinates(coordinates: any) {
+    this.markerCoordinates = coordinates;
+  }
+
+  addCircle() {
+    if (this.markerCoordinates === undefined) {
+      return;
+    }
+    const circle = leaflet.circle([this.markerCoordinates.altitude, this.markerCoordinates.latitude], {
+      color: '#bd1c1c',
```

Comment:
Sprawdzałeś może, czy nie da się pobrać kolorów z naszej palety?

Reply:
Sprawdziłem i można: https://stackoverflow.com/questions/40418804/access-sass-values-colors-from-variables-scss-in-typescript-angular2-ionic2 
Tylko nie wiem czy jest sens robić helpery do sass'a żeby dostarczyć dosłownie dwie wartości do jednego komponentu, wątpie żebyśmy z tego później korzystali.

Reply:
jakbyście mieli kolory w css variables to mozna byloby je wyciagnac tutaj